src/modules/app-registry.spec.js

```diff
@@ -0,0 +1,43 @@
+import reducer, {
+  appRegistryActivated,
+  appRegistryEmit,
+  APP_REGISTRY_ACTIVATED
+} from 'modules/app-registry';
+
+describe('app registry module', () => {
+  describe('#appRegistryActivated', () => {
+    it('returns the APP_REGISTRY_ACTIVATED action', () => {
+      expect(appRegistryActivated('app')).to.deep.equal({
+        type: APP_REGISTRY_ACTIVATED,
+        appRegistry: 'app'
+      });
+    });
+  });
+
+  describe('#appRegistryEmit', () => {
+    const spy = sinon.spy();
+    const appRegistry = { emit: spy };
+    const getState = () => {
+      return { appRegistry: appRegistry };
+    };
```

Comment:
Also can be shorter `const getState = () => { appRegistry };`
You can just ignore all my comments here since they just how I used to do write code. I share it with you only because of our code style conversation :)